pkg/datapath/loader/base.go

```diff
@@ -163,6 +164,18 @@ func addENIRules(sysSettings []sysctl.Setting, nodeAddressing types.NodeAddressi
 		IP:   nodeAddressing.IPv4().Router(),
 		Mask: net.CIDRMask(32, 32),
 	}
+
+	cidrs2 := make([]*net.IPNet, 0, 0)
+	for _, cidr := range cidrs {
+		cidrs2 = append(cidrs2, &cidr)
```

Comment:
Taking the `&cidr` address here looks like a bug: https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable

Reply:
Fixed bug according with suggestion from link. Thank you.